desktop/cmp/store/StoreFilterField.js

```diff
@@ -25,11 +25,20 @@ export class StoreFilterField extends Component {
         /** Store to filter */
         store: PT.instanceOf(BaseStore).isRequired,
         /** Names of fields in store's records to filter by */
-        fields: PT.arrayOf(PT.string).isRequired
+        fields: PT.arrayOf(PT.string).isRequired,
+        /** Delay (in ms) used to buffer filtering of the store after the value changes from user input (default is 200ms). Set to 0 to filter immediately after user input. */
+        filterBuffer: PT.number
     };
 
     @setter @observable value = '';
 
+    constructor(props) {
+        super(props);
+
+        const {filterBuffer = 200} = this.props;
```

Comment:
awesome syntax here.  Seems like this should be used everywhere we currently do 
>>filterBuffer = this.props.filterBuffer || 200;
It cleanly avoids any issues of 'falsy' values not getting set....

Reply:
Yes definitely, been trying to make sure I use it (and lodash `isNil()`) whenever appropriate after I introduced a couple bugs in Apollo from implicit boolean conversions to falsy values..